app/resources/cy/translation/payment/status.json

```diff
@@ -1,9 +1,8 @@
 {
   "title": "Talu a chyflwyno - statws",
-  "question": "Diolch yn fawr",
+  "headingNoPayment": "No payment required",
+  "headingPayment": "Payment received",
   "paragraph1": "Rydym wedi cael eich taliad a&rsquo;ch cais.",
```

Comment:
I know we aren't doing the welsh one but thinking this should match the english one, missing
""paragraph1": "You'll usually get the grant of representation within 16 weeks of submitting your application.","